azafea/event_processors/endless/metrics/request.py

```diff
@@ -34,6 +34,13 @@ class Request(Base):
     send_number = Column(Integer, nullable=False)
 
 
+class RequestByDay(View):
+    __tablename__ = 'metrics_request_v2_by_day'
```

Comment:
May I bikeshed the name of this view? It's not a table of requests by day. It's a table of machine IDs by day. :)

Reply:
Perhaps `machine_ids_by_day`?

I could imagine a future iteration of this would want to handle the case where an offline machine comes online and submits 5 days' worth of events. We'd want the machine to be represented on all 5 days, not just the day it happens to be online. To do that we'd back the view by some other table (perhaps uptime). If the name is more generic, we won't need to rename it & update queries to do that.

Reply:
> Perhaps `machine_ids_by_day`?

Much better.

> If the name is more generic, we won't need to rename it & update queries to do that.

I agree.

Reply:
Done!